README.md

```diff
@@ -1,2 +1,62 @@
-# platform-custom-scalars
-Library of customized GraphQL scalars for Ontotext Platform
+# Ontotext Platform Custom GraphQL Scalars
+A library providing customized scalars used in Ontotext Platform, written in JavaScript
```

Comment:
Maybe it's worth mentioning that these scalars are useful when you are writing an app using apollo federation where the platform is one of the services. The platform implements some custom scalars which are not known in the gateway and so in the client app you can't query fields from that type.

Reply:
uhmm I now saw you have described this in the purpose section at the end. Isn't it better to move that section in the beginning of the readme instead? :)

Reply:
Yeah, you are right. I'll move it.